concent_api/core/payments/base.py

```diff
@@ -64,3 +64,8 @@ def is_provider_account_status_positive(backend, request = None):
 @_add_backend
 def calculate_amount_pending(backend):
     return backend.calculate_amount_pending()
+
+
+@_add_backend
+def is_requestor_account_status_positive(backend, request):  # TODO use real concent_rpc when available
+    return backend.is_requestor_account_status_positive(request)
```

Comment:
We need to ensure that someone remembers to come back and add this. If you can't implement something because a part you depend on is missing I think you should add an issue to actually use that part when it's finally implemented. Or at least add it in the description of the issue that adds this part.

Reply:
done;
issue: https://github.com/golemfactory/concent/issues/255